hooks/useChannelDetails.ts

```diff
@@ -15,37 +17,66 @@ const DEFAULT_CHANNEL_DETAILS = {
   },
 };
 
+const DEFAULT_LOADING = {
+  info: true,
+  balance: true,
+  moderators: true,
+  topSong: true,
+  staking: true,
+  userStakedAmount: true,
```

Comment:
this feels like a lot of variables just to track whether a hook is loading.

how can we simplify this to a single, boolean, variable?